src/app/components/elements/cards/EventCard.tsx

```diff
@@ -30,45 +44,59 @@ export const EventCard = ({ event, pod = false }: { event: AugmentedEvent; pod?:
       )}
       <RS.CardBody className="d-flex flex-column">
         {title && (
-          <RS.CardTitle tag="h3">
+          <RS.CardTitle tag="h3" data-testid="event-card-title">
             {title}
+            {isPrivateEvent && (
+              <>
+                <br />
```

Comment:
`br`'s purpose is to add a line break in a block of text. You're misusing it here for styling/layout purposes. This is unfortunately a common thing, but it's incorrect. It's akin to using tables for layout. The right way to do this is through CSS.

Reply:
I've changed this by giving Private Event badge its own Row from RS